Security: Add disabled option for cookie samesite attribute (#21472)
grafana/grafana: docs/sources/installation/upgrading.md

## Upgrading to v6.6

The Generic OAuth setting `send_client_credentials_via_post`, used for supporting non-compliant providers, has been removed. From now on, Grafana will automatically detect if credentials should be sent as part of the URL or request body for a specific provider. The result will be remembered and used for additional OAuth requests for that provider.

### Important changes regarding SameSite cookie attribute

> The Stable version of Chrome 80 will treat cookies as `SameSite=Lax` by default if no `SameSite` attribute is specified, see https://www.chromestatus.com/feature/5088147346030592.

Reviewer: ```suggestion
Chrome 80 treats cookies as `SameSite=Lax` by default if no `SameSite` attribute is specified, see https://www.chromestatus.com/feature/5088147346030592.
```

> Due to this change in Chrome the `[security]` setting `cookie_samesite` configured to `none` now renders cookies with `SameSite=None` attribute compared to before where no `SameSite` attribute was added to cookies. To get the old behavior use value `disabled` instead of `none`, see [cookie_samesite]({{< relref "configuration/#cookie-samesite" >}}) for more information.

Reviewer: ```suggestion
Due to this change in Chrome, the `[security]` setting `cookie_samesite` configured to `none` now renders cookies with `SameSite=None` attribute compared to before where no `SameSite` attribute was added to cookies. To get the old behavior, use value `disabled` instead of `none`, see [cookie_samesite in Configuration]({{< relref "configuration/#cookie-samesite" >}}) for more information.
```

> *Note:* There is currently a bug affecting Mac OSX and iOS which causes `SameSite=None` cookies to be inadvertently treated as `SameSite=Strict` and therefore not sent with cross-site requests, see https://bugs.webkit.org/show_bug.cgi?id=198181. Until this is fixed, `SameSite=None` may not work properly on Safari.

Reviewer: ```suggestion
**Note:** There is currently a bug affecting Mac OSX and iOS that causes `SameSite=None` cookies to be treated as `SameSite=Strict` and therefore not sent with cross-site requests. (See https://bugs.webkit.org/show_bug.cgi?id=198181.) Until this is fixed, `SameSite=None` might not work properly on Safari.
```

> This version of Chrome will also reject insecure `SameSite=None` cookies, see https://www.chromestatus.com/feature/5633521622188032 for more information. Make sure that you

Reviewer: ```suggestion
This version of Chrome also rejects insecure `SameSite=None` cookies. See https://www.chromestatus.com/feature/5633521622188032 for more information. Make sure that you
```
